### ISSUE TITLE
Migrate to Vert.x 5

### ISSUE THREAD

**Issue by tsegismont** [vertx-howtos/fido2-webauthn-howto#3]
@FroMage can you take care of this one?

**Comment by FroMage** [comment]
For webauthn4j you mean?

**Comment by tsegismont** [comment]
Exactly

**Comment by tsegismont** [comment]
And upgrading to Vert.x 5 at the same time. I can assist you for this

**Comment by FroMage** [comment]
Is it normal that https://how-to.vertx.io/fido2-webauthn-howto/ returns a 404?

**Comment by FroMage** [comment]
Do you want to update this howto from vertx-auth-webauthn  to vertx-auth-webauthn4j, or create another repo for a new webauthn4j howto?

**Comment by tsegismont** [comment]
> Is it normal that https://how-to.vertx.io/fido2-webauthn-howto/ returns a 404?

No. I fixed that problem in https://github.com/vertx-howtos/fido2-webauthn-howto/commit/9dd2f9bc74852e252420e448b3220c9188b4e124

> Do you want to update this howto from vertx-auth-webauthn to vertx-auth-webauthn4j, or create another repo for a new webauthn4j howto?

I don't have any preference. Which option is more convenient for you?
